### PR TITLE
Initial improvements to code base

### PR DESCRIPTION
**Pull request by tkdchen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.idea/
+.env/
+__pycache__/
```
Comment (lcarva) [inline .gitignore]: Let's add a newline at the end of the file.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max_line_length = 100
```
Comment (lcarva) [inline setup.cfg]: Let's add a new line to end of file.

Comment (tkdchen) [reply setup.cfg]: Added newlines to both of these files.

**Comment by tkdchen** [comment]
Minor comments are fixed. I'm just going to merge this pr quickly. Thanks both of you. :)

**Pull request merged**